helm/observability-operator/files/alertmanager/url-template.tmpl.helm-template

```diff
@@ -0,0 +1,18 @@
+{{`
+{{ define "__alerturl" }}
+`}}{{ .alertmanager.grafanaAddress }}{{`/alerting/Mimir/{{ .CommonLabels.alertname }}/find
```

Comment:
I think we should be able to replace this with the grafanaExploreURL or this queryFromGeneratorURL instead https://grafana.com/docs/mimir/latest/references/architecture/components/alertmanager/#templating right?

Reply:
Yes I'll look into this later

Reply:
I'm trying to see if we can get rid of this whole file. that would make things simpler imo